pkgs/servers/x11/xorg/builder.sh

```diff
@@ -37,5 +37,6 @@ fi
 
```

Comment:
totally unrelated to this PR but I wouldn't mind if someone could remove this legacy builder script and put the content into mkDerivation like normally.